td_core/mdcount/ref_words_bot.py

```
@@ -0,0 +1,89 @@
+
+def make_old_values(all_data, lead_data):
+    # ---
+    # list for titles in both all_data and lead_data
+    old_values = list(set(all_data.keys()) & set(lead_data.keys()))
+    # ---
+    # remove duplicates from list
+    old_values = list(set(old_values))
+    # ---
+    list_ma = [x for x in old_values if (x in all_data and (x in lead_data))]
+    # ---
+    list_ma.extend([x for x in all_data.keys() if (x not in old_values and x.lower().startswith("video:"))])
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)